index.html

```diff
@@ -91,6 +92,45 @@ <h1>Avatar</h1>
             </iframe>
           </div>
         </div>
+        <!-- Avatar end -->
+
+        <!-- Badges start-->
+        <div class="container">
+          <h1>Badges</h1>
+          <p>Here we created custome classes for different types of Badges.</p>
+          <div class="inline">
+            <div class="badge-wrapper">
+              <i class="far fa-heart"></i>
+              <span class="badge-icon-val">3</span>
+            </div>
+            <div class="badge-wrapper">
+              <button>Action <span class="badge-icon-val">3</span></button>
+            </div>
+            <div class="badge-wrapper avatar-l">
+              <img src="./assets/avatar.png" alt="avatar" />
+              <span class="badge-img">3</span>
+            </div>
+            <div class="badge-wrapper avatar-l">
+              <img src="./assets/avatar.png" alt="avatar-status" />
+              <span class="badge-stat"></span>
+            </div>
+          </div>
+          <div class="code-container">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28221%2C231%2C233%2C1%29&t=seti&wt=none&l=auto&width=714&ds=true&dsyoff=20px&dsblur=66px&wc=true&wa=false&pv=43px&ph=55px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522far%2520fa-heart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-icon-val%2522%253E3%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cbutton%253EAction%253Cspan%2520class%253D%2522badge-icon-val%2522%253E3%253C%252Fspan%253E%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2520avatar-l%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-img%2522%253E3%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2520avatar-l%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-stat%2522%253E%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E"
+              style="
+                width: 714px;
```

Comment:
tip : you can change `iframe` width : 100%  to make it more responsive .

Reply:
@tanishk15sharma  Thank you for the suggestion. made changes.